app/src/main/java/com/fluffybros/tujefe/ui/home/HomeRecyclerAdapter.kt

```diff
@@ -5,7 +5,7 @@ import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
 import com.fluffybros.tujefe.databinding.HomeRecyclerItemBinding
 
-class HomeRecyclerAdapter(private val itemList: List<HomeRecyclerItem>) : RecyclerView.Adapter<HomeRecyclerAdapter.HomeViewHolder>() {
```

Comment:
let's undo all the changes in this file - the way you had it before was better

Reply:
If I don't define it as an ArrayList, I get a type mismatch error